egs2/qasr_tts/tts1/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=22050
+n_fft=1024
+n_shift=256
+opts=
+if [ "${fs}" -eq 48000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
+
+train_set=tr_no_dev
+valid_set=dev
+test_sets="dev eval1"
+
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# g2p=g2p_en # Include word separator
+g2p=g2p_en_no_space # Include no word separator
```

Comment:
you are using english G2P for arabic?

Reply:
I used the character based not the phoneme so it doesn't matter.

Reply:
than, I would suggest to set none here to ease the confusion.